rest_framework/versioning.py

```diff
@@ -74,7 +74,17 @@ class URLPathVersioning(BaseVersioning):
     invalid_version_message = _('Invalid version in URL path.')
 
     def determine_version(self, request, *args, **kwargs):
+        """
+        Use Default Version If 'version' Did Not Sets by Client in URL (So, It's Equal to None).
+        e.g:
+        api/users/ > Use Default Version (As Often The Default Version Is The Latest One).
+        api/v1/users/ > Check 'v1' Is Allowed?
+        """
```

Comment:
On blanance I think the intent of the function is clearer from the code itself than it would be from adding this docstring.

Reply:
You right, it's unnecessary.